Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoISE.yaml

```diff
@@ -0,0 +1,81 @@
+Parser:
```

Comment:
@spsocprime Changes looks good. Please format query to re-organize code

Reply:
@spsocprime Can you please take a look?

Reply:
formatted query